storescp/src/main.rs

```diff
@@ -300,6 +328,47 @@ fn create_cstore_response(
     obj
 }
 
+fn create_cecho_response(message_id: u16) -> InMemDicomObject<StandardDataDictionary> {
+    let mut obj = InMemDicomObject::new_empty();
```

Comment:
[`InMemDicomObject::from_element_iter`](https://docs.rs/dicom-object/latest/dicom_object/mem/struct.InMemDicomObject.html#method.from_element_iter) with an array would also work nicely, and provides a size hint.